cnctcli/actions/products/configuration_values_sync.py

```diff
@@ -0,0 +1,123 @@
+from collections import namedtuple
+
+from tqdm import trange
+
+from cnctcli.actions.products.sync import ProductSynchronizer
+from cnctcli.actions.products.constants import (
+    CONFIGURATION_HEADERS
+)
+
+import re
+import json
+
+fields = (v.replace(' ', '_').lower() for v in CONFIGURATION_HEADERS.values())
+
+_RowData = namedtuple('RowData', fields)
+
+
+class ConfigurationValuesSynchronizer(ProductSynchronizer):
+
+    def sync(self):
+        ws = self._wb['Configuration']
+        errors = {}
+        skipped_count = 0
+        created_items = []
+        updated_items = []
+        deleted_items = []
+
+        row_indexes = trange(2, ws.max_row + 1, position=0, disable=self._silent)
+        for row_idx in row_indexes:
+            data = _RowData(*[ws.cell(row_idx, col_idx).value for col_idx in range(1, 10)])
+            row_indexes.set_description(f'Processing Configuration value {data.id}')
+            if data.action == '-':
+                skipped_count += 1
+                continue
+            row_errors = self._validate_row(data)
+            if row_errors:
+                errors[row_idx] = row_errors
+                continue
+
+            scope_calc = data.id.split('#')
+            payload = {
+                'parameter': {
+                    'id': scope_calc[0]
+                }
+            }
+            if scope_calc[1]:
+                payload['item'] = {
+                    'id': scope_calc[1]
+                }
+            if scope_calc[2]:
+                payload['marketplace'] = {
+                    'id': scope_calc[2]
+                }
+
+            if data.action == 'update':
+                try:
+                    value = json.loads(data.value)
+                    payload['structured_value'] = value
+                except Exception:
```

Comment:
Better to catch JSONDecodeError

Reply:
as discussed, due nature of excel that can handle cell as many types when is "general"